cloudinit/distros/freebsd.py

```diff
@@ -192,5 +192,29 @@ def update_package_sources(self):
             freq=PER_INSTANCE,
         )
 
-
-# vi: ts=4 expandtab
+    def is_container(self) -> bool:
+        jailed, _ = subp.subp(["sysctl", "-n", "security.jail.jailed"])
+        if jailed.strip() == "0":
+            return False
+        return True
+
+    def virtual(self) -> str:
```

Comment:
I think `virtual()` could be better named `virtualization()`. or `virtualisation()`?
whatever the canonical form

which I guess is a pretty good argument for calling it `virtual()`